_includes/header.html

```diff
@@ -1,7 +1,26 @@
 <header>
   <nav class="navbar navbar-expand-md navbar-dark bg-primary flex-column flex-md-row">
     <div class="container">
-      <a class="navbar-brand" href="{{ site.baseurl }}/">{{ site.title }}</a>
+      <a class="navbar-brand" href="{{ site.baseurl }}/">
+        <svg
```

Comment:
Could we keep the SVG as a separate file and import/include it here? Then use CSS to change the colour and size.

Reply:
Sure, we did it that way for the [large version](https://github.com/dita-ot/website/blob/master/images/dita-ot-logo.svg) based on the original [dita-ot/logo](https://github.com/dita-ot/logo) files, so might make sense to do the same here.

(The approach used here came from the https://github.com/infotexture/dita-bootstrap/pull/11, where icons need to be embedded in template code rather than linked from source assets — but that constraint does not apply here.)

The `fill="currentColor"` obviates the need for color changes in CSS, but it might be handy to store this as a separate asset anyway in case we ever need it in different sizes for other things.

While we're at it, we may as well optimize the large version with something like [SVGO](https://github.com/svg/svgo), as it still contains bits of Inkscape/sodipodi cruft.

Reply:
> Could we keep the SVG as a separate file and import/include it here? Then use CSS to change the colour and size.

@jelovirt Experimenting with this today, it seems easier to style SVGs with CSS  if they're **_included_** in the HTML code.

https://css-tricks.com/the-many-ways-to-change-an-svg-fill-on-hover-and-when-to-use-them/
> SVGs can also be referenced using an `<img>` tag or as a background image. This allows the images to be cached and we can avoid bloating your HTML with chunks of SVG code. But the downside is a big one: we no longer have the ability to manipulate those properties using CSS.

With the SVG element in the HTML file, it's easy to override the fill with rules like this:

```scss 
.navbar-brand {
  path {
    fill: teal;

    @include hover-focus {
      fill: red;
    }
  }
}
```

I wasn't able to do the same when referencing the SVG in an `<img>` element.

The article outlines several workarounds, but I'm not sure it's worth the extra effort, since it works as is right now, and the before/after Lighthouse scores don't seem to indicate major performance penalties.

Are you aware of another method I've not considered?

Otherwise, I'd be inclined to merge this as is.

Reply:
@infotexture I meant that in Git the SVG is a separate file but then use some magic include markup in the HTML to embed the SVG into the final HTML during Jekyll build. Not that SVG would be referenced in the HTML with `<img>`.